### PR TITLE
Incremented to a newer version of play as a dependency. Added badges …

### PR DESCRIPTION
**Pull request by joelsonoda**
…for travis and codecov.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,3 +14,5 @@ cache:
 branches:
   only:
     - master
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (graingert) [inline .travis.yml]: trailing new line.

You should configure your editor to add these automatically

Comment (joelsonoda) [inline .travis.yml]: Done.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 scamandrill
 ===========
 
+[![Build Status](https://travis-ci.org/scamandrill/scamandrill.svg?branch=master)](https://travis-ci.org/scamandrill/scamandrill)
```
Comment (graingert) [inline README.md]: Can you use https://img.shields.io/travis/USER/REPO.svg etc for consistent badge appearance?

Comment (joelsonoda) [inline README.md]: Done

**Comment by graingert** [comment]
looks good, squash rebase etc then merge.

**Pull request merged**